src/components/AssetInfo/TokenSideBarContent.tsx

```diff
@@ -78,19 +99,42 @@ const TokenSideBarContent = ({
     }
   };
 
-  const approveChangeBeneficiary = () => {
-    const { approvedBeneficiary } = fieldValue;
-    handleFormActions(endorseTransfer, TOKEN_ACTION_TYPES.ENDORSE_BENEFICIARY, approvedBeneficiary);
+  const deployEscrowContractAction = async () => {
```

Comment:
should you be doing this here?

Reply:
its just used to toggle local state of loaders and error while calling the the deployContract service.